static/usr/share/man/man1/vdsm-client.1.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: 2022 Red Hat, Inc.
+SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
You can use troff comment (\") to include the license in the source file.

Trying to handle all special cases in same commit is not a good idea. I will be better to handle one special cases in every commit.